src/app/auth/guards/authorized.guard.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { CanActivate, CanLoad, Router, UrlTree } from '@angular/router';
+import { Observable } from 'rxjs';
+import { AuthService } from '../services/auth.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuthorizedGuard implements CanActivate {
```

Comment:
Do you implement `not-authorized.guard.ts`?

Reply:
added